### ISSUE TITLE
Uncharted: Golden Abyss black screen while loading game

### ISSUE THREAD

**Issue by Slo2018** [Vita3K/Vita3K-Android#398]
When i start game on my Android phone (MediaTek Helio G85, Android 12) i have black screen while loading. I'm using latest emulator version.

I tried on second phone with Exynos 9820 and Mali-G76 MP12.

Both phones have Mali GPU. Maybe is this the problem?

**Comment by suraj1234suraj** [comment]
Just install vita3k android 7 version and it will work properly. Mine is running really well after I installed an older version of vita3k

**Comment by starstremer** [comment]
Yeah but now you can’t get rid of the overlay to touch the screen to play :(
